dirty.html

```diff
@@ -1,205 +1,156 @@
-                  <div>Heroku</div>
-                  <div>Mandrill</div>
-                  <div>Twilio</div>
-                  <div>Exceptions</div>
-                  <div>Exception Handling</div>
-                  <div>Logs</div>
-                </div>
-              </div>
-            </div>
-            <div>
-              <div>
-                <div>
-                  <div>Cancan</div>
-                  <div>Carrierwave</div>
-                  <div>Bcrypt</div>
-                  <div>Devise</div>
-                  <div>Sidekiq</div>
-                  <div>Textacular</div>
-                  <div>Factory Girl</div>
-                  <div>High Voltage</div>
-                  <div>Rspec</div>
-                  <div>Nokogiri</div>
-                  <div>Acts as Draggable</div>
-                  <div>Acts as Taggable</div>
-                  <div>Will Paginate</div>
-                  <div>Nested Forms</div>
-                  <div>Geocoder</div>
-                  <div>Jekyll</div>
-                  <div>Better Errors</div>
-                </div>
-              </div>
-            </div>
-            <div>
-              <div>
-                <div>
-                  <div>Javascript</div>
-                  <div>jQuery</div>
-                  <div>Coffeescript</div>
-                  <div>JSON</div>
-                  <div>AJAX</div>
-                  <div>Ember</div>
-                  <div>Jasmine</div>
-                  <div>Underscore.js</div>                    
-                  <div>Templates</div>
-                  <div>Responsive design</div>
-                  <div>HTML</div>
-                  <div>HAML</div>
-                  <div>Handlebars</div>
-                  <div>ERB</div>                    
-                  <div>CSS</div>
-                  <div>SCSS, SASS</div>
-                  <div>Bootstrap</div>
-                </div>
-              </div>
-            </div>
+        <div class ="Students">
+          <h3>The Students</h3>
+            <h5>The London WDI Session 3 Coding Collective</h5> 
+            <p>We faced missing routes, broken links, aborted rake tasks, rejected mass-assignments, failed deployments, nilClasses, illegal nestings, conflicted merges, rougue debuggers, unexpected keyword ends, incomplete divs, unintentional infinite loops, uncooperative asset pipelines, empty migrations, unresponsive APIs, undefined defined methods, overly-sassy css, dirty data, faulty gems, terrible animated gifs, excessive pizza, punchy classmates, demanding instructors and long days. And we won!</p>
+        </div>    
+        <div class="concepts"></div>
+          <h3>The Concepts</h3>
+          <p>We were taught a lot of concepts. Over 12 weeks with three projects, one hackathon and not much sleep, we learned and practiced a full range of front- and back-end concepts. Take a look at some of what we covered... </p>
+          <div id="batch1">
+            <details>
```

Comment:
I think merging all these elements into one list may break a bit the structure they had in the original one... harder to follow right?